src/Reflection/ReflectionProperty.php

```diff
@@ -118,6 +119,33 @@ private function getVisibilityAsString()
     }
 
     /**
+     * Set the default visibility of this propery. Use the core \ReflectionProperty::IS_* values as parameters, e.g.:
+     *
+     * @param int $newVisibility
+     * @throws \InvalidArgumentException
+     */
+    public function setVisibility($newVisibility)
+    {
+        $this->node->type = ~Class_::MODIFIER_PRIVATE;
```

Comment:
they're unsetting those bits, then we set the new visibility bit instead; bitwise isn't my strong point